### DISCUSSION TITLE
Diagrams

### DISCUSSION THREAD

**Discussion by karyanmae** [Azure/missionlz#1065] [Q&A] [answered]
May I have the png diagrams in Visio formats? Especially the main one (hub and spokes) main/docs/images/20211220_updated_missionlz_as_of_Apr2024_light.png and src/bicep/add-ons/active-directory-domain-services/iaas/diagram.png. Thanks so much.

Honeyhline

**Comment by jamasten** [comment, marked as answer]
I'll email you what I have. Best, Jason
